README.md

```diff
@@ -1,70 +1,50 @@
 Copyright © 2020, TepacheLabs
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Songbox
 
-## Available Scripts
+Songbox is a web service that allows users to listen to their music stored in Dropbox without having to install any app, directly from a web interface.
 
-In the project directory, you can run:
+## Features:
 
-### `npm start`
+1. Out of the box solution for mobile devices.
+2. Dropbox stream cross-platform player.
+3. Play music, mp3, audiobooks from Dropbox.
+4. Supported audio formats: mp3, wav, ogg.
+5. No need to install Dropbox app.
+6. Access to Dropbox audio content anywhere
+7. All your music, songs, audio, audio books, audio reports in one place.
 
-Runs the app in the development mode.<br />
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+**Other:**
 
-The page will reload if you make edits.<br />
-You will also see any lint errors in the console.
+1. Using HTML5 web browser.
+2. No need to install any additional software.
+3. No stored files in system memory.
+4. Free to use.
 
-### `npm test`
+## Webapp Installation
 
-Launches the test runner in the interactive watch mode.<br />
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+```shell script
+$ git clone https://github.com/tepachelabs/songbox.webapp.git
+$ cd songbox.webapp
+$ npm install
+$ npm start
+# localhost:3000
+```
 
-### `npm run build`
+## API
```

Comment:
I don't think the API endpoints are relevant for this repo.

Reply:
do we have API documentation @ImAriel2D ? that's enough and this can be removed.